### PR TITLE
changed get-lib-dirs to get-lib-search-dirs

### PR DESCRIPTION
**Pull request by spdegabrielle**
@racket[get-lib-dirs] does not exist in docs.
-replaced with get-lib-search-dirs

https://docs.racket-lang.org/search/index.html?q=get-lib-dirs

https://docs.racket-lang.org/raco/dirs.html?q=get-lib-dirs#%28def._%28%28lib._setup%2Fdirs..rkt%29._get-lib-search-dirs%29%29

### CODE REVIEW

**Comment by rmculpepper** [comment]
This diff moves the corrected term somewhere else rather than fixing it in place.

**Comment by spdegabrielle** [comment]
my apologies - now fixed

**Comment by rmculpepper** [comment]
Oh, whoops, the original was correct: `get-lib-dirs` is an optional keyword argument whose default value is `get-lib-search-dirs`. But it would be good to change the prose to make that more apparent, because it's easy to miss or forget.

**Comment by spdegabrielle** [comment]
@rmculpepper thank you.  I have edited my PR to be clear that `#:get-lib-dirs` is an optional keyword, and state default value is `get-lib-search-dirs`.

File: pkgs/racket-doc/scribblings/foreign/libs.scrbl [hunk 1]
```diff
@@ -63,7 +63,8 @@ process:
 @itemlist[
 
  @item{If @racket[path] is not an absolute path, look in each
-       directory reported by @racket[get-lib-dirs]. In each
+       directories specified by the optional @racket[#:get-lib-dirs] keyword
+       (which defaults to @racket[get-lib-search-dirs]). In each
```
[the remark this answers is not in the document]
Comment (spdegabrielle) [reply pkgs/racket-doc/scribblings/foreign/libs.scrbl]: @bennn please make any changes you deem necessary

**Pull request merged**